NEWS

```diff
@@ -1,6 +1,22 @@
 master:
 * (port from 4.3) Improvement: the `URI adapter` now uses the content-disposition header to name the downloaded file.
 
+* Improvement: make the fingerprint digest configurable per attachment. The
+  default remains MD5 but this will change in a future version because it is
+  not considered secure anymore against intentional file corruption.
```

Comment:
Can you gather resources on this assertion, so we provide more information to paperclip users?